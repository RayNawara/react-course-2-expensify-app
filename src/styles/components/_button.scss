.button {
  display: inline-block;
  border: none;
  padding: $s-size;
  margin: 0;
  text-decoration: none;
  background: $blue;
  color: #ffffff;
  font-size: $font-size-large;
  font-weight: 300;
  cursor: pointer;
  line-height: 1;
  text-align: center;
  transition: background 250ms ease-in-out, 
              transform 150ms ease;
}

.button__link {
  background: transparent;
}

.button__secondary {
  background: #888888;
}