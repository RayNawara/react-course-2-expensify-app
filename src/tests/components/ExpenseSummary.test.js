import React from 'react';
import { shallow } from 'enzyme';
import { ExpenseSummary } from '../../components/ExpenseSummary';

test('should render expense list with expenses', () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={2} expenseTotal={943400} />);
  expect(wrapper).toMatchSnapshot();
});

test('should render expenseWord correctly when there is 1 expense', () => {
  const wrapper = shallow(<ExpenseSummary expenseCount={1} expenseTotal={9434001} />);
  expect(wrapper).toMatchSnapshot();
});