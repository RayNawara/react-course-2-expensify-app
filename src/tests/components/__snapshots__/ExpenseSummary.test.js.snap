// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list with expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totaling 
    $9,434.00
  </h1>
</div>
`;

exports[`should render expenseWord correctly when there is 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totaling 
    $94,340.01
  </h1>
</div>
`;
